games/s.yaml

```diff
@@ -342,7 +349,8 @@
 
 - name: SimTower
   meta:
-    genre: [simulation]
+    genre: [Simulation, Strategy]
```

Comment:
I've made an argument before that I don't think simulation games needs to be necessarily tagged as strategy either. In this case it would better be tagged as RTS too.